planning/behavior_velocity_planner/config/stop_line.param.yaml

```diff
@@ -4,5 +4,6 @@
       stop_margin: 0.0
       stop_check_dist: 2.0
       stop_duration_sec: 1.0
+      use_initialization_stop_line_state: false
```

Comment:
I think it is ok to true by default

Reply:
@yukkysaito 
Before this PR's change, it was fixed to false.
But I changed it to ros_param so that it can be changed for each project.
So I think false is fine...

Reply:
The behavior of stop_line module is more natural when his parameter is true, so I think it's no prolem to set it to True by default

Reply:
@yukkysaito @tkimura4 
I chenaged it to true.

Reply:
@tkimura4 thanks for the comment
@k-obitsu thank you